04.class/lib/memo.js

```diff
@@ -0,0 +1,35 @@
+import sqlite3 from "sqlite3";
+
+class Memo {
+  constructor() {}
+
+  createMemo() {
+    return new Promise((resolve) => {
+      this.db = new sqlite3.Database("./memo.sqlite3");
+      this.db.run("CREATE TABLE IF NOT EXISTS memos (memo TEXT)", () => {
+        resolve();
+      });
+    });
+  }
+
+  registerMemo(texts) {
+    this.db.run("INSERT INTO memos VALUES (?) ", texts.join("\r\n"));
```

Comment:
余分なスペースが含まれています。というのと、実引数が `this.input.contents` で仮引数が `texts` になっていて、名前が揃っていない感じがします。

```suggestion
    this.db.run("INSERT INTO memos VALUES (?)", texts.join("\r\n"));
```